packages/fxa-settings/src/pages/Signin/SigninTotpCode/mocks.tsx

```diff
@@ -2,4 +2,5 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+export const MOCK_EMAIL = 'test@example.com';
 export const MOCK_SERVICE = 'Example Service';
```

Comment:
We'll have to update this when the enum gets added in for relier parties but I don't think it has to be in this PR (or necessarily should be, depending on when it gets merged!)